tests/fixtures/markdown/export/expected.md

```diff
@@ -54,37 +63,53 @@ Max data processing per day: 10 TiB
 ## Service levels
 ### Availability
 *The server is available during support hours*
+
 - **percentage:** 99.9%
 
 ### Retention
 *Data is retained for one year*
+
 - **period:** P1Y
 
 ### Latency
 *Data is available within 25 hours after the order was placed*
+
 - **threshold:** 25h
+
 - **sourceTimestampField:** orders.order_timestamp
+
 - **processedTimestampField:** orders.processed_timestamp
 
 ### Freshness
 *The age of the youngest row in a table.*
+
 - **threshold:** 25h
+
 - **timestampField:** orders.order_timestamp
 
 ### Frequency
 *Data is delivered once a day*
+
 - **type:** batch
+
 - **interval:** daily
+
 - **cron:** 0 0 * * *
 
 ### Support
 *The data is available during typical business hours at headquarters*
+
 - **time:** 9am to 5pm in EST on business days
+
 - **responseTime:** 1h
 
 ### Backup
 *Data is backed up once a week, every Sunday at 0:00 UTC.*
+
```

Comment:
this empty line is fine, but I do not expect empty lines within the list. (between interval and cron, for example)